tssc/step_implementers/package/maven.py

```diff
@@ -84,15 +84,12 @@ def _run_step(self, runtime_step_config):
           'target')):
             if any(filename.endswith(ext) for ext in java_artifact_extenstions):
                 java_packaged_artifacts.append(filename)
-
+        if len(java_packaged_artifacts) > 1: # pragma: no cover
+            raise ValueError('pom resulted in multiple artifacts, this is unsupported')
```

Comment:
where is the test case that tests this exception?

Reply:
Resolved in new push